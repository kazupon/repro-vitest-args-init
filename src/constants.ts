export const SOMETHING_VALUES = {
  defaultLocale: 'en',
}