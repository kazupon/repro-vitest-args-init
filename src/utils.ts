import { SOMETHING_VALUES } from './constants'

export function getLocale({ defaultLocale = SOMETHING_VALUES.defaultLocale }: { defaultLocale?: string } = {}) {
  return defaultLocale
}