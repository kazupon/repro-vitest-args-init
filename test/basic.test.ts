import { test, assert, expect } from 'vitest'
import { getLocale } from '../src/utils'

test('1 + 1', () => {
  assert.equal(1 + 1, 2)
  expect({ foo: 1 }).toMatchSnapshot('foo')
})

test('getLocale', () => {
  assert.equal(getLocale(), 'en')
})