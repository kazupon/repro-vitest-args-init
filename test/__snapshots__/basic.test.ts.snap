// Vitest Snapshot v1

exports[`1 + 1 > foo 1`] = `
{
  "foo": 1,
}
`;
